spec/chromium-spec.js

```diff
@@ -937,6 +937,23 @@ describe('chromium feature', () => {
   })
 
   describe('storage', () => {
+    describe('DOM storage quota override', () => {
+      ['localStorage', 'sessionStorage'].forEach((storageName) => {
```

Comment:
Btw, you could have listed storages here, not their names.

```js
for (const storage of [localStorage, sessionStorage]) {
  <...>
}
```

Reply:
Indeed, I started with that, but I wanted to associate the name with the test (e.g. in the string parameter to `it(...)`) and didn't see a better way to do that (e.g. `localStorage.toString()` just returns
`"[object Storage]"`). If you like I can use an array of objects like `{name: 'localStorage', storage: localStorage}`.

Reply:
Oh, right, you use the names to build a test name. That's perfectly fine then.